code/src/components/Form.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
+import {RadioForm} from 'components/Radio'
+import {Text} from 'components/Text'
+
+export const values = {name: "", room: "", city: "", parking: "no"}
```

Comment:
If it's a yes/no answer maybe a boolean will suit good here?
```suggestion
export const values = {name: "", room: "", city: "", parking: false}
```